client/src/helpers/stardust/preExpandedConfig.ts

```diff
@@ -90,6 +104,30 @@ export function getInputsPreExpandedConfig(inputs: IInput[], unlocks: Unlock[],
     return inputsPreExpandedConfig;
 }
 
+function getReferencedAddresses(
+    inputs: IInput[],
+    unlocks: Unlock[],
```

Comment:
Since we always just use one unlock inside this function, consider just passing an `unlock` instead of `unlocks` and `idx` in the function. (+ optionally rename function to `getReferencedAddressesFromUnlock`)